packages/connectivity/connectivity/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: connectivity
 description: Flutter plugin for discovering the state of the network (WiFi &
   mobile/cellular) connectivity on Android and iOS.
 homepage: https://github.com/flutter/plugins/tree/master/packages/connectivity
```

Comment:
homepage is now: https://github.com/flutter/plugins/tree/master/packages/connectivity/connectivity